pkg/client/vcwallet/client.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vcwallet
```

Comment:
why `vcwallet`?

Reply:
All non-core interfaces are not in the spec anymore, like `storage : add, remove, get`.
Other functions like, `export, import, lock, unlock, verifyRaw, signRaw` are now optional.

Previously I insisted on calling it as universal wallet since our `add` function implementation was supposed to support all data models. From discussion [here](https://github.com/w3c-ccg/universal-wallet-interop-spec/issues/46) generating/adding/removing data models related to wallet contents will be completely controlled by wallet specific interface functions. And, storage interface functions are moved out of spec since they will be wallet implementation specific just like, VDR, KMS etc.

For example, VC wallet doesn't have to implement `verifyRaw, SignRaw` from the spec since VC wallet issue/prove/verify functions are already verifying proofs (In PR [spec#54](https://github.com/w3c-ccg/universal-wallet-interop-spec/pull/54) verifyRaw & signRaw functions are made optional).